tbcrawler/utils.py

```diff
@@ -32,6 +32,14 @@ def create_dir(dir_path):
     return dir_path
 
 
+def clone_dir_temporary(dir_path):
+    """Copy a folder to the same directory and append a timestamp."""
```

Comment:
Let's update this docstring,  it doesn't really append a timestamp and it doesn't "copy a folder to the same directory".